app/policies/chat_channel_policy.rb

```diff
@@ -51,6 +51,10 @@ def joining_invitation_response?
     record.present?
   end
 
+  def create_channel?
+    record.present? && user.id
```

Comment:
**suggestion (blocking):** As dicussed yesterday, let's change this to mods/trusted users for now.